docs/operator/helm-getting-started.md

```diff
@@ -0,0 +1,477 @@
++++
+title = "Grafana Agent Operator Helm Quickstart"
+weight = 110
++++
+# Grafana Agent Operator Helm Quickstart
+
+In this guide you'll learn how to deploy the [Grafana Agent Operator]({{< relref "./_index.md" >}}) into your Kubernetes cluster using the [grafana-agent-operator Helm chart](https://github.com/grafana/helm-charts/tree/main/charts/agent-operator). 
+
+You'll then deploy the following custom resources (CRs):
+
+- A `GrafanaAgent` resource, which discovers one or more `MetricsInstance` and `LogsInstances` resources.
+- A `MetricsInstance` resource that defines where to ship collected metrics. Under the hood, this rolls out a Grafana Agent StatefulSet that will scrape and ship metrics to a `remote_write` endpoint.
+- A `ServiceMonitor` resource to collect cAdvisor and kubelet metrics. Under the hood, this configures the `MetricsInstance` / Agent StatefulSet.
+- A `LogsInstance` resource that defines where to ship collected logs. Under the hood, this rolls out a Grafana Agent DaemonSet that will tail log files on your cluster nodes.
+- A `PodLogs` resource to collect container logs from Kubernetes Pods. Under the hood, this configures the`LogsInstance` / Agent DaemonSet.
+
+To learn more about the custom resources Operator provides, please consult [Operator architecture]({{< relref "./architecture.md" >}}).
```

Comment:
Do we want to explain why the operator doesn't deploy any of the custom resources?